### PR TITLE
Update CI-PHP

### PR DESCRIPTION
**Pull request by szepeviktor**
Goes into `add-missing-tests`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   quality:
-    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@e35bf8e686fbbbd05ecc8c056ff3487b1b95f865
+    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@d6f1253ff160f53f17135ff21e12d760318c7d5c
```
Comment (PrinsFrank) [inline .github/workflows/ci.yml:15]:
```suggestion
    uses: prinsfrank/CI-PHP/.github/workflows/quality.yml@main
```

This should be main and always use the latest version, but I ran into some caching in actions where it was not using the latest version without forcing an updated commit hash. Any idea how we could get that to work?

Comment (szepeviktor) [reply .github/workflows/ci.yml:15]: Yes. Go back 🪖 to hard hat mode.

1. make releases in CI-PHP
2. use Dependabot where that workflow is used

Comment (szepeviktor) [reply .github/workflows/ci.yml:15]: BTW many actions do _moving tags_
e.g. `actions/checkout@v4`
https://github.com/actions/checkout/tags
`v4` is overwritten at each release to point to the latest one.

👴🏻 I am not so daring.

**Comment by PrinsFrank** [comment]
@szepeviktor Thanks for the info on releases for actions! I created a v1 release, so I'll close this PR! Thanks again!

**Pull request closed without merging**